x-pack/plugins/ml/public/application/jobs/jobs_list/components/confirm_modals/managed_jobs_warning_callout.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiCallOut, EuiSpacer } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React from 'react';
+
+export const ManagedJobsWarningCallout = ({
+  jobsCount,
+  action,
+}: {
+  jobsCount: number;
+  action: string;
+}) => {
+  return (
+    <>
+      <EuiSpacer size="s" />
+      <EuiCallOut color="warning">
+        <FormattedMessage
+          id="xpack.ml.managedJobsWarningCallout"
+          defaultMessage="{jobsCount, plural, one {This job} other {At least one of these jobs}} is preconfigured by Elastic; {action} {jobsCount, plural, one {it} other {them}} might impact other parts of the product."
```

Comment:
The text needs to be different for the `starting` action - you should use the text shown in the screenshot which includes `starting it with a specific end time` when the 'Specify end time' option is shown.

![image](https://user-images.githubusercontent.com/7405507/148988844-969bdd7a-33f4-4657-8d1b-31c859cc5f7f.png)

Reply:
Fixed here https://github.com/elastic/kibana/pull/122305/commits/20dce3a9b564e47a2e19e9c81514ef2ae525d70b